hrpc/get.go

```diff
@@ -156,3 +194,20 @@ func familiesToColumn(families map[string][]string) []*pb.Column {
 	}
 	return cols
 }
+
+// Consistency is a Scan or Get option that requests the given
+// consistency of data.
+func Consistency(consistency QueryConsistency) func(Call) error {
```

Comment:
You can then move this to `query.go` (you will find similar function there)

Reply:
Done.